src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -131,6 +132,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 } else {
                     $this->hardCopy($originDir, $targetDir);
                 }
+                $validAssetDirs[] = $targetDir;
+            }
+        }
+        // Check in $bundlesDir, if all links/folder still have an existing Bundle
```

Comment:
This comment explains "how it does" something, but I'd recommend to explain "what it does" instead. Something like this:

```
// remove the assets of the bundles that no longer exist
```